library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/mixin/client/Class7196Mixin.java

```diff
@@ -32,12 +32,12 @@
 @Mixin(class_7196.class)
 public class Class7196Mixin {
 	@Inject(method = "method_41900", at = @At("HEAD"))
-	private void onStartDataPackLoad(class_7237.class_7238 arg, class_7237.class_6907<SaveProperties> arg2, CallbackInfoReturnable<WorldStem> cir) {
+	private void onStartDataPackLoad(C_kjxfcecs.C_nrmvgbka c_nrmvgbka, C_kjxfcecs.class_6907<SaveProperties> arg, CallbackInfoReturnable<WorldStem> cir) {
 		ResourceLoaderEvents.START_DATA_PACK_RELOAD.invoker().onStartDataPackReload(null, null);
 	}
 
 	@Inject(method = "method_41900", at = @At("RETURN"))
-	private void onEndDataPackLoad(class_7237.class_7238 arg, class_7237.class_6907<SaveProperties> arg2, CallbackInfoReturnable<WorldStem> cir) {
+	private void onEndDataPackLoad(C_kjxfcecs.C_nrmvgbka c_nrmvgbka, C_kjxfcecs.class_6907<SaveProperties> arg, CallbackInfoReturnable<WorldStem> cir) {
```

Comment:
It feels like something is wrong here, why is one of these inner classes in hashes and the other in intermediary

Reply:
Cracking question, I'll have a look into it

Reply:
If something is hashed it means it has one of its members mapped.
Meaning the second inner class could not have stuff mapped in it, creating this inconsistency :P

Reply:
Ah ok, that's good, because Linkie is choking and dying when I try to convert mappings at the moment 👎🏼